config/nginx.conf.erb

```diff
@@ -27,15 +40,20 @@ http {
 
     server {
         listen <%= ENV["PORT"] %>;
+        server_name micromasters.herokuapp.com;
+        location / {
+            return 301 https://micromasters.mit.edu$request_uri;
+        }
+    }
+
+    server {
+        listen <%= ENV["PORT"] %> default_server;
         server_name _;
         root /app;
 
-        if ($host = 'micromasters.herokuapp.com') {
-            return 301 'https://micromasters.mit.edu';
-        }
-
-        location ^~ /static/(.*$) {
-            try_files $uri $uri/ staticfiles/$1$args staticfiles/$1/$args =404;
+        location ~* /static/(.*$) {
+            expires max;
+            try_files $uri $uri/ staticfiles/$1 staticfiles/$1/ =404;
```

Comment:
FYI I found that I needed `staticfiles` to start with a slash, `/staticfiles/$1` for example